packages/dso-toolkit/src/styles/components/_list-button.scss

```diff
@@ -1,18 +1,21 @@
+.dso-list-button,
+.dso-btn-group {
+  + .dso-list-button,
+  + .dso-btn-group {
```

Comment:
Dit kan toch ook met
```
.dso-list-button,
.dso-btn-group {
  + & {
  }
}
```
?

Reply:
Dat kan, maar alleen als we 100% zeker zijn dat deze file geen andere context krijg doormiddel van een import oid.

Reply:
Goed argument wat mij betreft @daverhelst, maar... we houden ons op tal van plekken toch al niet aan dat uitgangspunt.

Reply:
Besproken: we doen het zonder de `&`